### DISCUSSION TITLE
Question: In the Pro plan, how am I charged per commiter?

### DISCUSSION THREAD

**Discussion by dgokcin** [vercel/community#962] [Help] [answered]
Hello

I understand that if I use the pro plan, I am charged 20 USDs. I am trying to understand the relationship between a GitHub user and a vercel active user. If I have 5 users in a GitHub repository that deploys to vercel in the pro plan how much will I be charged?

Thanks!

**Comment by swarnava** [comment, marked as answer]
For 5 users on GitHub that will deploy to Vercel, you would need 5 Vercel users as well which will cost you $20 x 5 = 100$.

**Reply by dgokcin** [reply]
@swarnava thanks for the reply. what do you mean by "users that will deploy to vercel"? does that exclude bot-users? what happens if I just deploy to vercel with a github actions bot and not use preview environments? would I still be charged 100 usds?

**Reply by swarnava** [reply]
Bot doesn't consider as actual user. So you can use GitHub action (without our native Git Integration). In that case, you won't need to add the comitter to Vercel.

**Reply by dgokcin** [reply]
great thanks!
